src/charm.py

```diff
@@ -236,6 +236,21 @@ def set_pod_spec(self, event):
                                 ],
                             }
                         ],
+                        "kubernetes": {
+                            "livenessProbe": {
+                                "initialDelaySeconds": 15,
+                                "httpGet": {
+                                    "path": "/",
+                                    "port": 9090,
```

Comment:
should this be the value of config `http-port` rather than hardcoded?
Or I would rather suggest to define a propery `http_port` for the charm class and re-use it here and in `containerPort` especially that we might deprecate the config in the future so in that case we'd change one line only.